syft/execution/translation/abstract.py

```diff
@@ -5,4 +5,7 @@ def __init__(self, plan):
         self.plan = plan
```

Comment:
Since it's an abstract base class, this should inherit from `ABC` (`from abc import ABC`.)